dependencyManagement/build.gradle.kts

```diff
@@ -11,10 +11,10 @@ data class DependencySet(val group: String, val version: String, val modules: Li
 val dependencyVersions = hashMapOf<String, String>()
 rootProject.extra["versions"] = dependencyVersions
 
-val otelVersion = "1.15.0"
+val otelVersion = "1.16.0"
 // IMPORTANT when updating opentelemetry instrumentation version, be sure to update bytebuddy version to match
-val otelInstrumentationVersion = "1.15.0"
-val otelInstrumentationAlphaVersion = "1.15.0-alpha"
+val otelInstrumentationVersion = "1.16.0"
+val otelInstrumentationAlphaVersion = "1.16.0-alpha"
 val otelContribAlphaVersion = "1.15.0-alpha"
```

Comment:
@jeanbisutti I'm prepping the 1.16.0 contrib release now, should be released tomorrow